domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParserTest.java

```diff
@@ -33,17 +34,24 @@ public void testInValidShowCondition() throws InvalidShowConditionException {
 
     }
 
+    @Test
+    void testAndConditions() throws InvalidShowConditionException {
+        ShowCondition sc = classUnderTest.parseShowCondition("field1= \"ABC AND XYZ\"  .AND. field2=\"some value\" ");
+
+        assertThat(sc.getShowConditionExpression(), is("field1=\"ABC AND XYZ\" .AND. field2=\"some value\""));
+        assertThat(sc.getFields(), hasItems("field1", "field2"));
+    }
+
```

Comment:
is  ```".AND. field2=\"some value\" "``` a valid show condition that the user can enter? in that case I've seen that there's no exception thrown but we are not able detect the fields correctly. Which could cause bugs.
Should we fix this ?

Reply:
Now it will throw an exception